.gitignore

```diff
@@ -178,6 +178,7 @@ pages/modifiers.js
 pages/internal-*
 pages/ShortDictionary.txt
 pages/LongDictionary.txt
+pages/theseus.json
```

Comment:
This should be marked as a generated file, not ignored (probably, CC @lyxal )

Reply:
Alright, I'll do that later. It didn't exist earlier so I assumed it wasn't to be tracked

Reply:
Done. I don't think there's any problem with keeping it in the repo. You should probably check that everything still works on the Theseus side, though, since the description and overloads have been modified, and there's a new keywords field